integrationtest/src/test/resources/springTest/src/test/java/org/mapstruct/itest/spring/SpringBasedMapperTest.java

```diff
@@ -45,6 +46,9 @@
     @Autowired
     private SourceTargetMapper mapper;
 
+    @Autowired
+    @Qualifier( "sourceTargetMapperDecorator" )
+    private DecoratedSourceTargetMapper decoratedMapper;
```

Comment:
Is there a way to have the decorated mapper injected without a qualifier (and let the wrapped mapper be injected into the decorator using a qualifier)?